dataland-frontend/src/assets/content.json

```diff
@@ -0,0 +1,205 @@
+{
+    "pages": [
+        {
+            "title": "Landing Page",
+            "url": "/",
+            "sections": [
+                {
+                    "title": "Welcome to Dataland",
+                    "text": [
+                        "This is the main introduction of our homepage."
+                    ],
+                    "image": [
+                        "/static/logos/gfx_logo_dataland_orange_S.svg"
+                    ]
+                },
+                {
+                    "title": "Intro",
+                    "text": [
+                        "Liberate Data - ",
+                        "Empower Autonomy.  ",
+                        "The alternative to data monopolies."
+                    ],
+                    "image": [
+                        "/static/logos/Dataland_D_svg_Logo.svg"
+                    ],
+                    "cards": [
+                        {
+                            "icon": "/static/icons/Search.svg",
+                            "text": "Search for a company to view or request data..."
+                        }
+                    ]
+                },
+                {
+                    "title": "Quotes",
+                    "text": [
+                        "START YOUR DATALAND JOURNEY"
+                    ],
+                    "cards": [
+                        {
+                            "icon": "wjiR2exsHCs",
+                            "title": "Christoph Benner,",
+                            "text": " CEO of Chom Capital",
+                            "date": "\"Join Dataland: it doesn't cost anything, it's open source and an entrusted structure.\""
+                        },
+                        {
+                            "icon": "BzUkAtdcKvg",
+                            "title": "Sven Schubert,",
+                            "text": " CEO of Envoria",
+                            "date": "\"Dataland is the only platform we know that is open to everyone and based on a non-profit business model.\""
+                        },
+                        {
+                            "icon": "9T9Bga8oojA",
+                            "title": "Christian Heller,",
+                            "text": " CEO of Value Balancing Alliance",
+                            "date": "\"We trust and believe in Dataland to provide better data to assess corporate impacts.\""
```

Comment:
Why does the field say "date" and there is a quote behind it?

Plus:

It is impossible to me in the frontend to see this text.
I think the videos are out of sync with the quote text underneath.
E.g. at the beginning the slide of Sven Schubert is in focus, but the quote from Christoph Benner is displayed:

![image](https://github.com/d-fine/Dataland/assets/99255296/74570a90-41c6-44fc-b055-bf899b2bc30b)

Reply:
as mentioned, theQuotes component is still in progress and not ready for review.
For the json content file, I am using available fields to not add specific ones only for this element, but I will definitely create ones on the next version.

Reply:
Ok.  Keeping this conversation open as reminder for myself